src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/annotatenuspec/addversioninfo.cmd

```diff
@@ -0,0 +1,30 @@
+@IF "%_echo%" NEQ "on" ECHO OFF
```

Comment:
We already have a target that gets the commit hash we should reuse that (https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L207) in fact we should even consider having the versioning.targets file generate this .version file.